README.md

```diff
@@ -1,13 +1,13 @@
+![Drupal Commerce](https://drupalcommerce.org/sites/default/files/dclogo_stacked_2c_on_white.png "Drupal Commerce logo")
+
 Drupal Commerce
 ===============
 [![Build Status](https://travis-ci.org/drupalcommerce/commerce.svg?branch=8.x-2.x)](https://travis-ci.org/drupalcommerce/commerce)
 
 Drupal Commerce is the leading flexible eCommerce solution for Drupal,
 powering over 60,000 online stores of all sizes.
 
-Note: 8.x-2.x is in beta. It is already usable for production websites 
-and upgrades between versions are supported. Bugs are still expected. 
-Please report bugs in the [issue queue](https://www.drupal.org/project/issues/commerce?version=8.x).
+Drupal Commerce 8.x-2.x is a stable release. It is usable for production websites.
```

Comment:
Actually, this part was wrong before your changes and after that. Just because what you're requesting to change is the `8.x-2.x` dev branch (sometimes called *default* or *main* or *master* or *HEAD*) of the Drupal Commerce. At the same time a **stable** release on the *Drupal* git repository it is not any particular branch. It is just some commit in the history having a tag, for example `8.x-2.1` (the latest currently). You'll not find this tag on the *Github* repository because it exists only on *Drupal* repository.

So, what we have: the users of the `HEAD` seeing the text which states that `8.x-2.x is in beta` which looks like a bit weird because **.x** versions can not be in `beta` or in `alpha` or `stable`. Furthermore, [this text is seen](http://cgit.drupalcode.org/commerce/tree/README.md?h=8.x-2.1#n8) by people using the latest `stable` version, not `beta`. The same with your text. You're stating that `Drupal Commerce 8.x-2.x is a stable release` and `It is usable for production websites`. Again, it is wrong.

I am not so fine in English but the right phrase should be found which will satisfy both users of the `dev` version and `stable` release of the *Drupal Commerce*.